### PR TITLE
Remove links to old install doc

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

A follow-up to #10995. There's already a redirect in place, but changing links to the new location is always preferred.

##### Component Name

area/docs

### CODE REVIEW

**Review by OdysLam — APPROVED** [review]

File: packaging/installer/README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ operating systems.
     os="General Linux with one-line installer (recommended)"
     svg="linux" />
   <InstallBox
-    to="#run-netdata-with-docker"
+    to="/docs/agent/packaging/installer/methods/docker"
```
Comment (ilyam8) [inline packaging/installer/README.md:81]: docker is not in [/docs/agent/packaging/installer/methods](https://github.com/netdata/netdata-learn-docusaurus/tree/master/docs/agent/packaging/installer/methods), what i am missing?

Comment (joelhans) [reply packaging/installer/README.md:81]: Whoops, my bad. Should have those fixed now.

Comment (ilyam8) [reply packaging/installer/README.md:81]: From what I see docker install method is not fixed

Comment (ilyam8) [reply packaging/installer/README.md:81]: ok ok, just noticed that

- docker
   - to: `/docs/agent/packaging/installer/methods/docker`
   - location: **not in** the `/docs/agent/packaging/installer/methods/`


- other methods
   - to: `/docs/agent/packaging/installer/methods/{name}`
   - location: **in** the `/docs/agent/packaging/installer/methods/` (macos, kubernetes, offline, etc.)

=> that is why I decided that docker method wasn't going to work.

**Review by OdysLam — APPROVED** [review]

File: packaging/installer/README.md (hunk 1 above)
Comment (joelhans) [reply packaging/installer/README.md]: Yup, you're right. It should have been `/docs/agent/packaging/docker` all along.

Comment (ilyam8) [reply packaging/installer/README.md]: cool, haha I was about to give up on this, decided to try one more time to explain it 😄

**Review by ilyam8 — APPROVED** [review]

**Review by OdysLam — APPROVED** [review]

**Pull request merged**